src/createNumbersGenerator.js

```diff
@@ -26,7 +26,22 @@
  * @return {number}
  */
 function createNumbersGenerator(numbers, min, max) {
-  // write code here
+  let filteredNumbers;
+  if (numbers !== undefined) {
+    if (min !== undefined && max !== undefined) {
```

Comment:
good, but if we will receive only `max` or `min`?

you can solve this problem with another check or add default values to min and max

Reply:
use the logic operator `!` and `!!` instead of `===` or `!==`